lib/SPIRV/libSPIRV/SPIRVModule.cpp

```diff
@@ -547,7 +547,10 @@ SPIRVModuleImpl::getCurrentLine() const {
 
 void SPIRVModuleImpl::setCurrentLine(
     const std::shared_ptr<const SPIRVLine> &Line) {
-  CurrentLine = Line;
+  if (Line)
```

Comment:
I assume this relates to a code analyzer report for 'explicit null dereference'. If so, can you please elaborate on how this could happen? Also, can the std::shared_ptr constructor even return null?